cmd/compose/main.go

```diff
@@ -58,15 +60,15 @@ func main() {
 		Http:    cl,
 	}
 
-	users := &usersapi.Client{
-		BaseURL: "http://localhost:8080",
-		Http:    cl,
-	}
+	conn, err := grpc.Dial("localhost:5050", grpc.WithTransportCredentials(insecure.NewCredentials()))
+	defer conn.Close()
```

Comment:
you missed the error check here.

Reply:
would be great to take a look at [dial options ](https://pkg.go.dev/google.golang.org/grpc#DialOption), for example to set timeouts.